doctest.sh

```diff
@@ -77,7 +77,7 @@ nfail=0
 cd tmp/doctest
 for file in */*.jou; do
     # print file and line number, as in "doc/foo.md:123: "
```

Comment:
```suggestion
    # Print file and line number, as in "doc/foo.md:123: "
    # Newline is deleted to avoid warning on NetBSD 9.3, see issue #500
```